src/content/api/cli.md

```diff
@@ -23,7 +23,9 @@ For proper usage and easy distribution of this configuration, webpack can be con
 
 Have a look at the [installation guide](/guides/installation) if you don't already have webpack installed.
 
-T> The new CLI for webpack is under development. New features are being added such as the `--init` flag. [Check it out!](https://github.com/webpack/webpack-cli)
+The new CLI for webpack is under development. New features are being added such as the `--init` flag. [Check it out!](https://github.com/webpack/webpack-cli)
+
+T> Recently a new lightweight, opinionated webpack CLI became available for use under [webpack-command](https://github.com/webpack-contrib/webpack-command).
```

Comment:
It somehow should mentioned that it is part of webpack-contrib, not just a third party library.